src/js/App.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div id="app">
```

Comment:
`#app` is a very generic name for an ID, which can only ever be used once on a page…

```suggestion
  <div id="rectangles-root">
```

Tho this is not essential to a demo maybe…

Reply:
changed the id to "rectangles-root"